doc/source/getting_started/using_julia.rst

```diff
@@ -0,0 +1,197 @@
+To create a virtual environment, use the ``activate`` command with the name of the new environment that you want to create or activate.
+
+.. code-block:: julia
+
+    (@1.7) pkg> activate julia_test
+      Activating project at `C:\Users\USER\julia_test`
+
+
+A message should appear, indicating that the new package (``julia_test``) has been activated. This environment name will now precede the command line.
+
+.. code-block:: julia
+
+    (julia_test) pkg>
+
+Next install the *PyCall* package by typing:
+
+.. code-block:: julia
+
+    (julia_test) pkg> add PyCall
+
+
+To use *PyCall*, press the backspace key to go to the Julia command line.
+The command line will now be precede by the name ``Julia``. 
+
+.. code-block:: julia
+
+    julia>
+
+Next use the *PyCall* package with:
+
+.. code-block:: julia
+
+    julia> using PyCall
+
+
+This should be enough to use packages included in a basic Python distribution. 
+
+
+For example:
+
+.. code-block:: julia
+
+    julia> math = pyimport("math")
+    math.sin(math.pi/4) # returns ≈ 1/√2 = 0.70710678..
+
+
+Installing PyMAPDL in Julia
+============================
+
+*PyCall* includes a lightweight Python environment that uses `Conda <https://conda.io>`_ to manage and access Python packages.
+This environment, currently based on Python 3.9.7, includes the standard basic Python libraries.
+However, because it is a fully working Python environment, you can still use it from outside the Julia command line and install Python packages using ``pip``.
+
+To install PyMAPDL, first locate the Python executable with:
+
+.. code-block:: julia
+
+    julia> PyCall.python
+    "C:\\Users\\USER\\.julia\\conda\\3\\python.exe"
+
+In Linux, the above code prints the following, where ``python3`` is the default Python3 installation for the operating system.
+
+.. code-block:: julia
+    
+    julia> PyCall.python
+    "python3"
+
+
+.. note::
+
+    In Linux, there are no specific installation steps. You only need to add the Julia executable to the path.
+    Hence,  Julia's Python installation path can differ from user to user.
+    For example, if you uncompress the source files in ``/home/USER/Julia``, Julia's path will be 
+    ``/home/USER/Julia/julia-1.7.2/bin``
+
+You would use this Python executable to install PyMAPDL:
+
+.. code:: bash
+
+    C:\Users\USER\.julia\conda\3\python.exe -m pip install ansys-mapdl-core
+
+In Linux:, you would install with:
+
+.. code:: bash
+
+    python3 -m pip install ansys-mapdl-core
+
+
+Finally, after restarting Julia, you can import PyMAPDL using the same procedure as described above:
+
+.. code-block:: julia
+    
+    julia> using PyCall
+    julia> pymapdl = pyimport("ansys.mapdl.core")
+    PyObject <module 'ansys.mapdl.core' from 'C:\\Users\\USER\\.julia\\conda\\3\\lib\\site-packages\\ansys\\mapdl\\core\\__init__.py'>
+    julia> print(mapdl.__str__())
```

Comment:
```suggestion
    julia> print(mapdl)
```

Reply:
I have to disagree on this:

```julia
julia> print(mapdl)
PyObject <ansys.mapdl.core.mapdl_grpc.MapdlGrpc object at 0x00000000667E6A60>
julia> print(mapdl.__str__())
Product:             Ansys Mechanical Enterprise
MAPDL Version:       21.2
ansys.mapdl Version: 0.60.6
```
I guess it makes sense as a way to show it is not a "native" julia object.

Reply:
Then this might be better:
```suggestion
    julia> print(str(mapdl))
```

Reply:
The ``string`` julia functions does still show the julia str implementation. The only way is to call the python method ``__str__``.